README.md

```diff
@@ -3,13 +3,18 @@
 ![Github Actions Status](https://github.com/silx-kit/jupyterlab-h5web/workflows/Build/badge.svg)
 [![PyPI version](https://badge.fury.io/py/jupyterlab-h5web.svg)](https://badge.fury.io/py/jupyterlab-h5web)
 
-A JupyterLab extension to explore and visualize HDF5 file contents. Based on
-[h5web](https://github.com/silx-kit/h5web).
+**This extension is still in active development. Please report any encountered
+issue.**
 
-**WARNING: This extension is still in active development. Please report any
-encountered issue.**
+![Demo](https://user-images.githubusercontent.com/2936402/114533096-d5e68000-9c4d-11eb-81d3-67d313c9216f.gif)
 
-![Extension screenshot](https://user-images.githubusercontent.com/42204205/106109102-6c280100-6149-11eb-96eb-38a14983702f.png)
+**jupyterlab-h5web** is a JupyterLab extension to explore and visualize HDF5
+file contents, using the web-based viewer
+[h5web](https://github.com/silx-kit/h5web). Visualisations are tailored for
+[NeXus](https://www.nexusformat.org/) but **jupyterlab-h5web** can be used to
+explore any HDF5 file.
```

Comment:
Sounds like we don't have generic visualizations, which we do. Maybe just say something like "H5Web supports the `NeXus` format."